alpha_automl/wrapper_primitives/huggingface.py

```diff
@@ -17,8 +20,7 @@ def fit(self, X, y=None):
         return self
 
     def transform(self, texts):
-
-        list_texts = [text for text in texts]  # since texts is in the form of numpy array
+        list_texts = [text if str(text)!='nan' else '' for text in texts]  # since texts is in the form of numpy array
```

Comment:
Why do we need this filter for 'nan' values? Is this specific to some dataset or numpy is generating this somewhere in alpha-automl?